protocol/messenger_settings.go

```diff
@@ -20,10 +20,18 @@ func (m *Messenger) SetLogLevel(request *requests.SetLogLevel) error {
 	return nodecfg.SetLogLevel(m.database, request.LogLevel)
 }
 
+func (m *Messenger) SetMaxLogBackups(request *requests.SetMaxLogBackups) error {
+	return nodecfg.SetMaxLogBackups(m.database, request.MaxLogBackups)
```

Comment:
I wonder if we should do some validations on the request first? Like make sure  `MaxLogBackups` is not empty and not smaller than 0?

We have a couple `Validate()` functions in the code to check the content of the `requests`. You can take inspiration (copy paste :smile: )

Reply:
Fixed